bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| factoryReset                     | general         | Factory reset                                                    | Switch                   |      | ON = reset start                                                                                     |
+| supplyAirFanStage                | operation       | Supply air fan stage                                             | Number                   |      | 0 - 4                                                                                                |
+| extractAirFanStage               | operation       | Extract air fan stage                                            | Number                   |      | 0 - 4                                                                                                |
+| fanStageStepped0to2v             | unitConfig      | Fan stage for stepped mode - range 0-2V                          | Number                   |      | 0 - 2                                                                                                |
+| fanStageStepped2to4v             | unitConfig      | Fan stage for stepped mode - range 2-4V                          | Number                   |      | 0 - 4                                                                                                |
+| fanStageStepped4to6v             | unitConfig      | Fan stage for stepped mode - range 4-6V                          | Number                   |      | 0 - 4                                                                                                |
+| fanStageStepped6to8v             | unitConfig      | Fan stage for stepped mode - range 6-8V                          | Number                   |      | 0 - 4                                                                                                |
+| fanStageStepped8to10v            | unitConfig      | Fan stage for stepped mode - range 8-10V                         | Number                   |      | 0 - 4                                                                                                |
+| offsetExtractAir                 | unitConfig      | Offset extract air                                               | Number                   |      | float                                                                                                |
+| assignmentFanStages              | unitConfig      | Assignment fan stages - stepped or 0-10V                         | Switch                   |      | OFF = 0-10V, ON = stepped                                                                            |
+| sensorNameHumidityAndTemp1       | humidityControl | Sensor name - humidity and temperature 1                         | String                   |      |                                                                                                      |

```

Comment:
Are the following names rather properties?

Reply:
If it's not an intended use case to change these parameters via OH, then I believe implementing them as properties will definitely make sense.